solarforecastarbiter/metrics/tests/test_preprocessing.py

```diff
@@ -0,0 +1,154 @@
+import time
+import datetime
+import numpy as np
+import pandas as pd
+import pytest
+
+from solarforecastarbiter import datamodel
+from solarforecastarbiter.metrics import preprocessing
+
+
+THREE_HOURS = pd.date_range(start='2019-03-31T12:00:00',
+                            periods=3,
+                            freq='60min',
+                            tz='MST',
+                            name='timestamp')
+
+THREE_HOUR_SERIES = pd.Series(np.arange(1., 4., 1.), index=THREE_HOURS,
+                              name='value')
+
+THIRTEEN_10MIN = pd.date_range(start='2019-03-31T12:00:00',
+                               periods=13,
+                               freq='10min',
+                               tz='MST',
+                               name='timestamp')
+
+THIRTEEN_10MIN_SERIES = pd.Series((np.arange(0., 13., 1.)/6)+1,
+                                  index=THIRTEEN_10MIN)
+
+# Bitwise-flag integers (only test validated and versioned data)
+CL_UF = int(0b10010)  # Cloudy and User Flagged (18)
```

Comment:
This looks like just Nighttime and version 0. `0b100011` would be cloudy, version 0, and user flagged

Reply:
Thanks. Updated